### PR TITLE
Validate if root device is encrypted prior to actual configuration

### PR DESCRIPTION
**Pull request by gobindadas**
Fixes: RHBZ- 1812351,1812327,1812322,1812299,1812301,1812072

### CODE REVIEW

File: playbooks/hc-ansible-deployment/tasks/luks_tang_setup.yml [hunk 1]
```diff
@@ -11,23 +11,20 @@
...
 
     - name: Check if root device is encrypted
-      assert:
-        that:
-          - "result != 0"
-        fail_msg: "The given {{ rootdevice }} is not encrypted."
-      when: when: gluster_infra_tangservers is defined
+      fail:
+        msg: "The given boot device {{ rootdevice }} is not encrypted."
+      when: "result.failed == true"
```
Comment (pkesavap) [inline playbooks/hc-ansible-deployment/tasks/luks_tang_setup.yml]: Apologies for my ignorance, but in line 15 we register a variable result that contain the output of shell command which was executed at line 14, and in line 25 we check if the output of shell command i.e, result.failed == true,  but in and instance what if the  result contains "Device <device-name> does not exist or access denied" , will this condition handle it?

Comment (gobindadas) [reply playbooks/hc-ansible-deployment/tasks/luks_tang_setup.yml]: If the device does not exist then also the commad will fail and in error the reason will be there.

Comment (pkesavap) [reply playbooks/hc-ansible-deployment/tasks/luks_tang_setup.yml]: oh cool i missed that

**Comment by pkesavap** [comment]
looks good to me

**Pull request merged**